backend/Lingva.BusinessLayer/Contracts/IMembersService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using Lingva.DataAccessLayer.Entities;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace Lingva.DataAccessLayer.Entities
+{
+    public interface IMembersService
+    {
+        
+        void Delete(int id);
+
+        EventMembers JoinGroup(int userID, int groupID);
```

Comment:
Add a new breakline.
We have got a deal to follow code convention together.
Between a property or method must be one breakline.